internal/epub/epub.go

```diff
@@ -255,6 +266,18 @@ func (e *ePub) Write() error {
 		if e.Image.Manga {
 			titleAlign = "right:0"
 		}
+		titleWidth := e.Image.View.Width
+		titleHeight := e.Image.View.Height
+		if e.Image.Orientation == "portrait" && len(part.Images) > 0 {
+			titleWidth = part.Images[0].Width
+			titleHeight = part.Images[0].Height
+		}
+		cssDimension := "%dpx"
+		cssViewPort := "width=%d,height=%d"
+		if e.Image.Orientation == "portrait" {
+			cssDimension = "unset"
```

Comment:
the viewport need to be set on fixed layout. 
you can use the image resolution here.

Reply:
You mean like I did for the title? `part.Images[0].Width`? Well it doesn't seem to work properly with the double pages if I set the width and height here. If I just leave it unset it seems to work perfectly fine.

If we enforce a fixed layout here then we would need to split the double page and add the option to not keep the original one, which is why I initially added the option. Or will this be fixed with mentioned changes to content opf and apple xml?


Then goal of this PR is to add an option to not have a border or margin around the rendered pages and simply render the pages in the dimensions / aspect ratio of the original file, effectively ignoring the view from the profile. If that can be achieved in another way then I'm happy to assist where I can.

Reply:
the aspect ratio of the original image is respected. I keep it as it is.
I just render in a viewBox, which will add margin if necessary.

Let say you have a image of 1:1.5 of ratio (1 width = 1.5 height), and you device have 1:1.33 of ratio, you see here that even if you use the viewport with maximum size, the device will add margin because the width/height ratio are not the same. The only way to remove any margin is to not respect the original ratio aspect, which will distord the image.
It's simply achievable by settings width/height to the deviceSize. But that's ugly.

I will try something about portait mode, let me see what happen on various device (scribe/oasis/kindle app on mac).

By the way, it seems the Kindle App on IOS and Mac sucks, they are not supported manga/spread in landscape/ ... So clearly far from the eInk devices, which support epub 3.3 version (spread left/center/right, and more).

For Tablet on IOS or Android, the best viewer I know is YACReader, which support CBZ/CBR directly, and have fantastic render without any size limitation.
This tools is really more for eInk device, and I can add option to support more type of device, but those option need to work on any device.

For example: Portrait Only is possible on any device, and we can ensure to support any orientation of the image with the proper settings.

I will create a branch, will try to pickup some of your patch, and go from there to support portrait on any device without the 1:1.6 ratio I apply to support landscape, and also removing the spreading which is useless in that case.